docs/methods/cardanoSignTransaction.md

```diff
@@ -1,46 +1,88 @@
 
 ## Cardano: Sign transaction
 Asks device to sign given transaction. User is asked to confirm all transaction
-details on Trezor.
+details on device.
+
+Stake pool opration transactions are explained in separate file.
```

Comment:
The original trezor docs already have docs for stake pool registration, I would bootstrap that "pool operations" docs file mentioned here in this PR and reference it here, so it's not accidentally left out